src/applications/survivors-benefits/config/chapters/03-military-history/nationalGuardServicePeriod.js

```diff
@@ -0,0 +1,37 @@
+import {
+  titleUI,
+  currentOrPastDateUI,
+  currentOrPastDateSchema,
+  textUI,
+  textSchema,
+  internationalPhoneUI,
+  internationalPhoneSchema,
+} from 'platform/forms-system/src/js/web-component-patterns';
+
+/** @type {PageSchema} */
+export default {
+  title: 'National Guard service period',
+  path: 'veteran/national-guard-service-period',
+  uiSchema: {
+    ...titleUI('National Guard service period'),
+    dateOfActivation: currentOrPastDateUI({
+      title: 'Date of activation',
+      monthSelect: false,
+    }),
+    unitName: textUI({
+      title: 'Reserve or National Guard Unit name',
+    }),
+    unitPhoneNumber: internationalPhoneUI({
+      title: 'Reserve or National Guard Unit primary phone number',
+    }),
+  },
+  schema: {
+    type: 'object',
+    required: ['dateOfActivation', 'unitPhoneNumber'],
+    properties: {
+      dateOfActivation: currentOrPastDateSchema,
+      unitName: textSchema,
+      unitPhoneNumber: internationalPhoneSchema(),
```

Comment:
To make this properly required, you need to change to `primaryPhone: internationalPhoneSchema({ required: true }),`

Reply:
Hmm did not work

Reply:
got it working